### PR TITLE
Skill - LadyBlade

### PR DESCRIPTION
**Pull request by JesterWizard**
If unit is female, double weapon might.

### CODE REVIEW

**Comment by MokhaLeee** [comment]
put them to common skill calc

File: Wizardry/Core/BattleSys/Source/PreBattleCalc.c [hunk 1]
```diff
@@ -20,6 +20,11 @@ void ComputeBattleUnitAttack(struct BattleUnit * attacker, struct BattleUnit * d
     int status;
     status = GetItemMight(attacker->weapon);
 
+#if (defined(SID_LadyBlade) && (COMMON_SKILL_VALID(SID_LadyBlade)))
+    if (!(UNIT_CATTRIBUTES(&attacker->unit) & CA_FEMALE))
```
Comment (MokhaLeee) [inline Wizardry/Core/BattleSys/Source/PreBattleCalc.c:24]: also this results in male

**Comment by JesterWizard** [comment]
> put them to common skill calc

Not sure exactly what you mean by this?

File: Wizardry/Core/BattleSys/Source/PreBattleCalc.c (hunk 1 above)
Comment (JesterWizard) [reply Wizardry/Core/BattleSys/Source/PreBattleCalc.c:24]: Ah true. I didn't noticed because Eirika had a Rapier and I was testing on Saar.

**Pull request merged**